packages/seacas/libraries/ioss/src/exodus/Ioex_Utils.C

```diff
@@ -616,7 +617,7 @@ namespace Ioex {
     // Try to decode an id from the name.
     std::string name_string = entity->get_property(prop_name).get_string();
     std::string type_name   = entity->short_type_string();
-    if (std::strncmp(type_name.c_str(), name_string.c_str(), type_name.size()) == 0) {
+    if (strncasecmp(type_name.c_str(), name_string.c_str(), type_name.size()) == 0) {
```

Comment:
Actually, `str_equal` is wrong since the match is whether `type_name` is substring of `name_string`...

Reply:
We could add a `Ioss::Utils::strncasecmp()` that is compatible across various platforms since it's not a std::c++ function ... unless we just leave it as the regular libc implementation